### PR TITLE
Activity notification inactive tabs

### PR DESCRIPTION
**Pull request by johanthejohan**
Adds notification for activity in inactive tabs.

### CODE REVIEW

**Comment by Eightquake** [comment]
Looks great! The notification isn't intrusive but very noticeable, just as it should.

I noticed however that the notification still shows up when the line that was added should be excluded by the filter - i.e with a filter set to `i` and the line `Another lone w/o that letter` is added, the green circle appears, but when switching to the tab the lines haven't changed.
I'm not sure how one would fix this though, would it require that the tabs has knowledge of the filter?

File: src/js/view/styledComponents/TabPanelStyledComponents.js [hunk 1]
```diff
@@ -45,3 +45,22 @@ export const Button = styled.div`
...
+export const Indicator = styled.div`
+  width: 10px;
+  height: 10px;
+  display: inline-block;
+  background-color: lightgreen;
+  border-radius: 50%;
+  position: relative;
+  top: -8px;
+  left: 5px;
+  border: darkgrey solid 1px;
```
Comment (Doverstav) [inline src/js/view/styledComponents/TabPanelStyledComponents.js]: Maybe darken this border just slightly? (Yes, in css colors grey is darker than darkgrey)
```suggestion
  border: grey solid 1px;
```

File: src/js/view/styledComponents/TabPanelStyledComponents.js [hunk 2]
```diff
@@ -45,3 +45,22 @@ export const Button = styled.div`
   padding: 5px;
   margin-left: 5px;
 `;
+
+export const Indicator = styled.div`
+  width: 10px;
+  height: 10px;
+  display: inline-block;
+  background-color: lightgreen;
```
Comment (Doverstav) [inline src/js/view/styledComponents/TabPanelStyledComponents.js]: Perhaps this color can be changed that stands out a bit more and is more attention-grabbing, like orange? Lightgreen feels a bit too muted, at least for me personally.
```suggestion
  background-color: orange;
```

File: src/js/view/styledComponents/TabPanelStyledComponents.js (hunk 1 above)
Comment (johanthejohan) [reply src/js/view/styledComponents/TabPanelStyledComponents.js]: I see your point, will darken the border.

File: src/js/view/styledComponents/TabPanelStyledComponents.js (hunk 2 above)
Comment (johanthejohan) [reply src/js/view/styledComponents/TabPanelStyledComponents.js]: Makes sense, will change color to something more alerting.

**Comment by johanthejohan** [comment]
I totally missed testing with the filter turned on @Eightquake, nicely spotted!
Sorting this out will probably take some time. At the moment the application keeps track of changes in file size. In order to sort this out we would have to start tracking new rows after filtering. 
So the question is should we merge this in as @Doverstav suggested and create a separate issue for this or should we keep this branch active?

**Comment by Eightquake** [comment]
I'd say this is ready to merge, as it is to show activity for a file and not that new lines are being shown. We could talk a bit more about it in tomorrow's meeting!

**Pull request merged**